bundles/AdminBundle/Helper/AssetTypeHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\AdminBundle\Helper;
+
+use Pimcore\Model\Asset;
+use Symfony\Component\Mime\MimeTypes;
+
+class AssetTypeHelper
```

Comment:
You should register this class as a service since autowire isn't working otherwise

Reply:
Are there any reasons to create separate classes for this instead of just inlining the methods into the listener?

Reply:
My idea was to make writing unit tests more easier.